src/js/messaging/config.js

```diff
@@ -9,7 +10,7 @@ function getHeaders(optionalHeaders = {}) {
     defaultHeaders.Authorization = `Token token=${localStorage.userToken}`;
   }
 
-  return Object.assign({}, defaultHeaders, optionalHeaders);
+  return assign({}, defaultHeaders, optionalHeaders);
```

Comment:
Since the FP functions have no side effects, I think you can just do without the `{}` in the first parameter wherever we use this form of `assign`. Whatever goes in the first parameter shouldn't mutate.